contentcuration/contentcuration/settings.py

```diff
@@ -86,6 +87,16 @@
     }
 }
 
+# READ-ONLY SETTINGS
+# Set STUDIO_INCIDENT_TYPE to a key from contentcuration.utils.incidents to activate
+INCIDENT = INCIDENTS.get(os.getenv('STUDIO_INCIDENT_TYPE'))
```

Comment:
can you move this os.getenv to another variable called `INCIDENT_TYPE`? Might be good to extract the real incident on some later stage.